source/_integrations/switch_as_x.markdown

```diff
@@ -43,3 +43,16 @@ Switch entity:
 Type:
   description: The type of entity you want the switch to become.
 {% endconfiguration_basic %}
+
+## Alternatives
+Other ways to set a Switch as X include:
+ - [Helper](#helper)
+ - [Entity Properties](#entity-properties)
+
+### Helper
+{% my helpers title="Settings -> Devices & Services -> Helpers" %}
+![Helper example](/images/integrations/switch_as_x/Helper_Switch_as_X.png)
+
+### Entity Properties
+{% my entities title="Settings -> Devices & Services -> Entities" %}
+![Entity properties exampl](/images/integrations/switch_as_x/Entities_Properties.png)
```

Comment:
Let's just make a single chapter on: Configuring switch as X from an existing entity as an alternative method or something (so we don't need all these paragraphs at all).

Reply:
I was struggling to fit the screenshots to the other ways of doing it but I will take another look later.

Reply:
Re-wrote this part along with the above comment to **Switch as X from existing entity**